packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -383,52 +379,31 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
   };
 
   /**
-   * Sets the slab thickness to all the volume actors in the viewport.
+   * It sets the slabThickness of the actors of the viewport. If actorUIDs are
+   * provided, it will only set the slabThickness of the actors with the given
+   * uid, but if actorUIDs are not provided, it will set the slabThickness of
+   * all the actors in the viewport.
    *
-   * @param slabThickness - The slab thickness to set.
+   * @param slabThickness - number -The slab thickness to set.
+   * @param actorUIDs - An array of actor UIDs. If this is not
```

Comment:
Hmmm, since this is actually a filter, shall we name it as such, at first glance its a bit weird that if you specify it only does it that one, otherwise it does it to _all_. In most other cases throughout the project, if you omit a uid, it applies it to the first item found.

Maybe `filterActorUIDs` or something?

Reply:
Propogate this comment to all other API of this format here.